joomla/joomla-cms
# Better descriptor of what the "Keyboard shortcut" shortcut does

Pull Request for Issue # .

### Summary of Changes
Changes "Keyboard Shortcuts" text shown at bottom of admin to "Show keyboard shortcuts"


### Testing Instructions
Check bottom of admin panel showing the "Keyboard Shortcuts"
apply patch
Check that it has changed.


### Actual result BEFORE applying this Pull Request

![image](https://user-images.githubusercontent.com/1850089/190054733-104dd7b3-0c4a-400d-a68a-91334f4fbfca.png)


### Expected result AFTER applying this Pull Request

![image](https://user-images.githubusercontent.com/1850089/190919544-db74b75b-f37c-4758-bfa0-88d0e8606964.png)



### Documentation Changes Required

if used then Update user documentation showing the change in verbiage.
